source/linux/Foundational_Components/Kernel/_Users_Guide.rst

```diff
@@ -728,18 +518,20 @@ Installing the Kernel Image and Device Tree Binaries
     .. code-block:: console
 
         cd <kernel sources dir>
-        sudo cp arch/arm/boot/zImage <rootfs path>/boot
-        sudo cp arch/arm/boot/dts/<dt file>.dtb <rootfs path>/boot/dtb
+        sudo cp arch/arm/boot/zImage $boot
+        sudo cp arch/arm/boot/dts/ti/omap/<dt file>.dtb $boot
 
     For example, if you wanted to copy the kernel image and BeagleBone
-    Black device tree file to the rootfs partition of a SD card you would
+    Black device tree file to the SD card partition, you would
     enter the below commands:
 
     .. code-block:: console
 
          cd <kernel sources dir>
-         sudo cp arch/arm/boot/zImage /media/rootfs/boot
-         arch/arm/boot/dts/am335x-boneblack.dtb /media/rootfs/boot
+         sudo cp arch/arm/boot/zImage $boot
+         arch/arm/boot/dts/ti/omap/am335x-boneblack.dtb $boot
```

Comment:
Isn't this incorrect now? Don't the legacy devices also use `$boot/dtb/ti` directory structures now?